service/user/src/test/kotlin/me/helloc/techwikiplus/user/infrastructure/logging/UseCaseLoggingIntegrationTest.kt

```
@@ -0,0 +1,74 @@
+
+    @Test
+    fun `예외 발생 시 ERROR 레벨로 로깅된다`() {
+        // given
+        val email = "duplicate@example.com"
+        val nickname = "duplicateuser"
+        val password = "Password123!"
+
+        // when & then - 예외가 발생해도 AOP 로깅이 동작함
+        try {
+            userSignUpUseCaseWrapper.signUp(email, nickname, password)
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)